tests/cupy_tests/lib_tests/test_strided_tricks.py

```diff
@@ -29,6 +29,65 @@ def test_rolling_window(self, xp):
         return a_rolling
 
 
+class TestSlidingWindowView:
+    def test_1d(self):
```

Comment:
Could you use `testing.numpy_cupy_array_equal()` to compare the outputs of `sliding_window_view` between NumPy and CuPy?

Reply:
`@testing.numpy_cupy_array_equal()` is added for all tests and is passing now...